packages/core/test/convert.spec.ts

```diff
@@ -20,11 +20,6 @@ import {
   sleep,
 } from '../src'
 
-// We filter out app chains because we expect the mainnet test to cover them adequately
-const fetchNonRollupStackChains = () => {
-  return SPHINX_NETWORKS.filter((network) => network.rollupStack === undefined)
-}
```

Comment:
This filters out Optimism, Arbitrum, and Polygon zkEVM, which probably shouldn't be filtered out because these stacks won't have coverage at all

Reply:
That's why this function was removed. It's no longer used to filter the networks at all.

Reply:
My bad, I guess i'm color blind